src/main/java/Event.java

```diff
@@ -0,0 +1,28 @@
+public class Event extends Task {
+    private String startTime;
+    private String endTime;
+
+    public Event(String name, String startTime, String endTime) {
+        super(name);
+
+        String[] startParts = startTime.split(" ");
+        StringBuilder startSB = new StringBuilder(startParts[0]).append(": ");
```

Comment:
startSb instead of startSB since it is an abbreviation?

Reply:
Thanks for pointing that out.